dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/ArtifactDownloader.java

```diff
@@ -297,14 +301,26 @@ private void configureProxyAuthentication() {
         }
     }
 
-    private void configureAuthentication(final MavenRepository repository) {
+    private void configureAuthentication(final MavenRepository repository) throws InstallException {
 
         if (repository.getUserId() != null && repository.getPassword() != null &&
             envMap.get("https.proxyUser") == null && envMap.get("http.proxyUser") == null) {
+            String decodedPwd = repository.getPassword();
+            try {
+                decodedPwd = PasswordUtil.decode(repository.getPassword());
+            } catch (InvalidPasswordDecodingException ipde) {
+                decodedPwd = repository.getPassword();
```

Comment:
I think it's too assumptive that if we get InvalidPasswordDecodingException that it's just something we can ignore and assume that it's an unencoded password. From the API: 
```
If the encoded_string is null or invalid. Or the decoded_string is null.
```

Also from the API: 
```
Decode the provided string. The string should consist of the algorithm to be used for decoding and encoded string. For example, {xor}CDo9Hgw=. Use this method unless the encryption key needs to be specified for the AES encryption. 

 An empty algorithm "{}" is treated as not encoded. However, a missing algorithm will trigger the InvalidPasswordDecodingException.
```
I think we're assuming if we see this exception that the user provided a plain text password without the `{}`. But we instead detect that case and add them before hand easily: 
```
String encodedPassword = repository.getPassword();
        if (!encodedPassword.startsWith("{")) {
            encodedPassword = "{}" + encodedPassword;
        }
```

Reply:
Maybe we can't check for `{}` if it's AES encrypted, in which case we can't add the `{}`. Not sure, we would need to test it.

Reply:
Maybe we need to use `passwordDecode()` to support AES? either way both show that the algorithm should be provided in between `{` and `}` and unencoded passwords should still have `{}` in front to signal they are not encoded.